metashare/api/admin.py

```diff
@@ -1,9 +1,15 @@
 from django.contrib import admin
 
-from .models import User
+from .models import Product, User
 
 
 @admin.register(User)
 class UserAdmin(admin.ModelAdmin):
     list_display = ("username", "is_active", "is_staff", "is_superuser", "date_joined")
     search_fields = ("username",)
+
+
+@admin.register(Product)
+class ProductAdmin(admin.ModelAdmin):
+    list_display = ("name", "description")
+    prepopulated_fields = {"repo_url": ("name",)}
```

Comment:
I'm not sure this still makes sense, now that it's a URL.

Reply:
Right, it doesn't.